client/client.go

```diff
@@ -453,6 +453,12 @@ type (
 	// WARNING: Worker versioning is currently experimental.
 	WorkerVersioningRules = internal.WorkerVersioningRules
 
+	// WorkflowUpdateRPCTimeoutOrCancelledError is an error that occurs when an update RPC call times out or is cancelled.
+	//
+	// Note, this is not related to any general concept of timing out or cancelling a running update, this is only related to the client call itself.
+	// NOTE: Experimental
+	WorkflowUpdateRPCTimeoutOrCancelledError = internal.WorkflowUpdateRPCTimeoutOrCancelledError
```

Comment:
If you would like, you can only use one `l`, e.g. `WorkflowUpdateRPCTimeoutOrCanceledError`, since that's what we do at https://pkg.go.dev/go.temporal.io/api/serviceerror#Canceled. Different languages do different things there.